ricecooker/classes/files.py

```diff
@@ -147,28 +72,38 @@ def write_and_get_hash(path, write_to_file, hash=None):
     except (MissingSchema, InvalidSchema):
         # If path is a local file path, try to open the file (generate hash if none provided)
         with open(path, 'rb') as fobj:
-            for chunk in iter(lambda: fobj.read(4096), b""):
+            for chunk in iter(lambda: fobj.read(2097152), b""):
                 write_to_file.write(chunk)
                 hash.update(chunk)
 
     assert write_to_file.tell() > 0, "File failed to write (corrupted)."
 
     return hash
 
+def copy_file_to_storage(filename, srcfile, delete_original=False):
+    # Some files might have been closed, so only filepath will work
+    if isinstance(srcfile, str):
+        srcfile = open(srcfile, 'rb')
+
+    # Write file to local storage
+    with open(config.get_storage_path(filename), 'wb') as destf:
+        shutil.copyfileobj(srcfile, destf)
+
+    if delete_original:
+        os.remove(srcfile.name)
```

Comment:
If `delete_original` is set, it'd be much more efficient to move the file, rather than copy and delete.